catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -94,50 +79,61 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
     [images]
   );
 
-  const imageRenderer = useCallback(galleryImage => {
-    const photo: GalleryImageProps = galleryImage.photo;
+  const imageRenderer: React.FC<RenderPhotoProps<GalleryImageProps>> = ({
```

Comment:
Think you could just use `FunctionComponent`, there was a moved to use it instead of `FC` + it's already imported!

Reply:
done